### ISSUE TITLE
Type to navigate feature defective since I upgraded to Firefox 97

### ISSUE THREAD

**Issue by mzso** [aaFn/Bookmark-search-plus-2#224]
Hi!
I upgraded from 95 to 97 and now it fully seems to be fully unresponsive for characters with diacritics.
No reaction to such keypresses, and also bookmarks/folders  can only be searched for until the first such character with a diacritic mark.
Searchfield search works normally at least.

I also see errors popping up in the console:
```js
12:27:02.840 Key event: keydown key: é char: undefined target: [object HTMLTableCellElement] classList: selbrow 5 [panel.js:5247:9](moz-extension://89e09938-37d0-44e4-8068-fce270d1d77a/sidebar/panel.js)
12:27:03.752 Key event: keydown key: Alt char: undefined target: [object HTMLTableCellElement] classList: selbrow [panel.js:5247:9](moz-extension://89e09938-37d0-44e4-8068-fce270d1d77a/sidebar/panel.js)
12:27:03.844 Error: Invalid window ID: -1 undefined
12:27:09.080 Key event: keydown key: é char: undefined target: [object HTMLTableCellElement] classList: selbrow 2 [panel.js:5247:9](moz-extension://89e09938-37d0-44e4-8068-fce270d1d77a/sidebar/panel.js)
12:27:10.417 Key event: keydown key: Alt char: undefined target: [object HTMLTableCellElement] classList: selbrow [panel.js:5247:9](moz-extension://89e09938-37d0-44e4-8068-fce270d1d77a/sidebar/panel.js)
12:27:10.863 Error: Invalid window ID: -1 undefined
```

**Comment by mzso** [comment]
[**@aaFn**](https://github.com/aaFn) commented on [2022. febr. 12. 23:48 CET](https://github.com/aaFn/Bookmark-search-plus-2/issues/224#issuecomment-1037546503 "2022-02-12T22:48:03Z - Replied by Github Reply Comments"):
> Hi [@mzso](https://github.com/mzso) , no, this is still something missing in my code not handling diacritic characters in input, only alphabetic ones (I erroneously thought that the native `\w` regex pattern was including diacritics).
> 
> I will correct it in next version (2.0.101).

I'm quite sure I used it successfully before. So maybe Mozilla changed something under this. Anyway, it's good to know that it'll be fixed.

**Comment by aaFn** [comment]
Corrected in 2.0.101 which is out tonight.

**Comment by mzso** [comment]
[**@aaFn**](https://github.com/aaFn) commented on [2022. febr. 16. 00:08 CET](https://github.com/aaFn/Bookmark-search-plus-2/issues/224#issuecomment-1040888229 "2022-02-15T23:08:25Z - Replied by Github Reply Comments"):
> Corrected in 2.0.101 which is out tonight.

Great! It seems to work now.
